Package.swift

```diff
@@ -509,7 +509,7 @@ let package = Package(
         ),
         .package(url: "https://github.com/tuist/Path", .upToNextMajor(from: "0.3.0")),
         .package(
-            url: "https://github.com/tuist/XcodeGraph.git", exact: "0.19.4"
+            url: "https://github.com/foyoodo/XcodeGraph.git", branch: "feature/mac-catalyst-with-macos-bundle"
```

Comment:
You can now update this to `1.1.0`: https://github.com/tuist/XcodeGraph/releases/tag/1.1.0

Reply:
Done! Thank you for taking the time to review this PR :)